### ISSUE TITLE
aursearch check() error when building with makepkg

### ISSUE THREAD

**Issue by emilhem** [AladW/aurutils#183]
When building aurutils with makepkg I get the error:
```
In bin/aursearch line 106:
    printf '%s\n'
           ^-- SC2183: This format string has 1 variables, but is passed 0 arguments.
```
I assume that this is a check that can be ignored?

**Comment by AladW** [comment]
This issue was fixed with 1.3.6, but I believe the AUR package has not been updated yet. Please try bumping pkgrel manually.

**Comment by AladW** [comment]
The AUR package was updated.
